src/cli/commands/add.js

```diff
@@ -125,7 +126,8 @@ module.exports = {
     hash: {
       type: 'string',
       choices: Object.keys(mh.names),
-      describe: 'Hash function to use. Will set CID version to 1 if used. (experimental)'
+      describe: 'Hash function to use. Will set CID version to 1 if used. (experimental)',
+      default: 'sha2-256'
```

Comment:
I was setting the default value to hashing function as `sha2-256`. This will set CID to v1.
But CID defaults to v0. And the tests that failed were looking up for v0 ID, which did not match.
Possible solutions:
- Fix tests to match with v1 ID.
- Remove default hash

@alanshaw Thoughts?

Reply:
Yes I'd remove the default for now